LDRH_choice.py

```diff
@@ -231,7 +247,14 @@ def importConditions(path):
         'Spatial': data.StairHandler(startVal = 150,
             stepType = 'db', stepSizes=[3,3,2,2,1,1],#[8,4,4,2,2,1,1], #reduce step size every two reversals
             minVal=0, maxVal=350, nUp=1, nDown=3,  #will home in on the 80% threshold
-            nTrials = 10)
+            nTrials = 10),
+        'posterior_matching': {
```

Comment:
But then we are defining handlers that we don't use later. We should just
conditionally define them from the start and not have to worry at any later
point which one we used.

On Mon, 10 Aug 2015 12:23 Ratnaganadi <notifications@github.com> wrote:

> In LDRH_choice.py
> <https://github.com/CHD-UCSD/REACH-Choice-Task/pull/29#discussion_r36673072>
> :
>
> > @@ -231,7 +247,14 @@ def importConditions(path):
> >          'Spatial': data.StairHandler(startVal = 150,
> >              stepType = 'db', stepSizes=[3,3,2,2,1,1],#[8,4,4,2,2,1,1], #reduce step size every two reversals
> >              minVal=0, maxVal=350, nUp=1, nDown=3,  #will home in on the 80% threshold
> > -            nTrials = 10)
> > +            nTrials = 10),
> > +        'posterior_matching': {
>
> I did this:
>
> if use_posterior_matching==False:
> all_handlers['staircasing']={...}
> this_handler = all_handlers['staircasing']
> elif use_posterior_matching==True:
> all_handlers['posterior_matching']={...}
> this_handler = all_handlers['posterior_matching']
>
> then, below, I just do
> if operation:
> handler = this_handler[task][operation]
> else:
> handler = this_handler[task]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CHD-UCSD/REACH-Choice-Task/pull/29/files#r36673072>.
>